.vscode/settings.json

```diff
@@ -96,6 +99,8 @@
     "CADL_VERBOSE_TEST_OUTPUT": "true",
     "NODE_OPTIONS": "--stack-trace-limit=50"
   },
+  "prettier.prettierPath": "./packages/compiler/node_modules/prettier",
+  "prettier.documentSelectors": ["**/*.cadl"],
```

Comment:
ohhh, that must have been what I was missing..., this doesn't affect other extensions that were already working?

Reply:
It still works for .ts, .md., and .yml at least.

I also had to put that parser override in prettierrc